CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## [0.58.17](https://github.com/brave/brave-browser/releases/tag/v0.58.17)
+## [0.58.18](https://github.com/brave/brave-browser/releases/tag/v0.58.18)
  
  - Added text snippets under the tip panel and tip banner when a publisher hasn't been verified. ([#2697](https://github.com/brave/brave-browser/issues/2697))
  - Removed all images for unverified publishers on Brave Rewards. ([#2696](https://github.com/brave/brave-browser/issues/2696))
+ - Removed YouTube favicons for unverified publishers in Tip table under Brave Rewards. ([#2715](https://github.com/brave/brave-browser/issues/2715))
```

Comment:
Tip should be tip no?

Reply:
sure, wanted to emphasis the `Tip` table but I guess it doesn't make much sense.